docs/releases.md

```diff
@@ -41,6 +41,33 @@ These changes and information only affect those users who directly follow the ma
 - **Annoying regressions already known:** This is a list of already known regressions to be fixed by the second part of the refactoring which will concentrate on the JSON API and front-end:
   - _Password-dialog is shown for non-existing and private albums:_ If a user requests a URL for a non-existing or private album, the password dialog will be presented to the user as if the album was password-protected. The password dialog will be presented repeatedly to the user independent of the provided password as the authentication will never succeed.
 
+_Addendum for those who migrated between 2022-01-13 and 2022-01-15:_
+
+After the initial merge to the master branch we became aware of some performance regressions due to missing indices on the DB.
+We have patched the migration script to create the indices.
+If you migrated to the head of master after 2022-01-15 you are fine.
+If you were an early adopter and migrated between 2022-01-13 (when the original merge came out) and 2022-01-15, the indices are missing.
+
+Unfortunately, the migration script can only add these indices while the affected table is being created, not after table creation.
+If you are affected, you have two options:
+
+ - Option 1: Re-run the migration
+ - Option 2: Add the missing indices manually
+
+For Option 1 (all DB engines): First rollback the original migration via `./artisan migrate:rollback`, update the master branch via `git pull` and re-run the migration again via `./artisan migrate`. As always make a backup of your DB first.
+
+For Option 2 (MySQL/PostgreSQL only): If you don't want to spend the time the migration takes, you use MySQL or PostgreSQL and you have access to your SQL console, you can also add the missing indices manually.
```

Comment:
It's not just the time. It's the extra risk of performing a rollback operation that may not have been as well tested as the forward migration, and the fact that the new IDs after re-run will be different than before, won't they?

Reply:
Totally right. Both aspects are important. I will try to rephrase it.

Reply:
Better now?